po/tr.po

```diff
@@ -1048,135 +1043,214 @@ msgid ""
 "io/github/stars/google/comprehensive-rust?style=flat-square)](https://github."
 "com/google/comprehensive-rust/stargazers)"
 msgstr ""
+"[![Build workflow](https://img.shields.io/github/actions/workflow/status/"
+"google/comprehensive-rust/build.yml?style=flat-square)](https://github.com/"
+"google/comprehensive-rust/actions/workflows/build.yml?query=branch%3Amain) [!"
+"[GitHub contributors](https://img.shields.io/github/contributors/google/"
+"comprehensive-rust?style=flat-square)](https://github.com/google/"
+"comprehensive-rust/graphs/contributors) [![GitHub stars](https://img.shields."
+"io/github/stars/google/comprehensive-rust?style=flat-square)](https://github."
+"com/google/comprehensive-rust/stargazers)"
 
 #: src/index.md:7
 msgid ""
 "This is a free Rust course developed by the Android team at Google. The "
 "course covers the full spectrum of Rust, from basic syntax to advanced "
 "topics like generics and error handling."
 msgstr ""
+"Bu, Google'daki Android ekibi tarafından geliştirilen ücretsiz bir Rust "
+"kursudur. Kurs, temel sözdiziminden jenerikler ve hata işleme gibi ileri "
+"düzey konulara kadar Rust'ın tüm yelpazesini kapsar."
 
 #: src/index.md:11
 msgid ""
+"The latest version of the course can be found at <https://google.github.io/"
+"comprehensive-rust/>. If you are reading somewhere else, please check there "
+"for updates."
+msgstr ""
+"Kursun en son sürümüne <https://google.github.io/comprehensive-rust/> "
+"adresinden ulaşılabilir. Başka bir yerde okuyorsanız, lütfen güncellemeleri "
+"kontrol edin."
+
+#: src/index.md:15
+msgid ""
 "The goal of the course is to teach you Rust. We assume you don't know "
 "anything about Rust and hope to:"
 msgstr ""
+"Kursun amacı size Rust'ı öğretmektir. Rust hakkında hiçbir şey bilmediğinizi "
+"varsayıyoruz ve şunu umuyoruz:"
 
-#: src/index.md:14
+#: src/index.md:18
 msgid "Give you a comprehensive understanding of the Rust syntax and language."
-msgstr ""
+msgstr "Rust sözdizimi ve dili hakkında kapsamlı bir anlayış sunmayı."
 
-#: src/index.md:15
+#: src/index.md:19
 msgid "Enable you to modify existing programs and write new programs in Rust."
 msgstr ""
+"Rust'ta mevcut programları değiştirmenize ve yeni programlar yazmanıza "
+"olanak sağlamayı."
```

Comment:
Thank you for these translations. I am nitpicking. Perhalps we can say "olanak saglar" to have a verb at the end.